sympy/matrices/expressions/tests/test_matexpr.py

```diff
@@ -357,6 +357,11 @@ def test_issue_2749():
     assert (A.T * A).I.as_explicit() == Matrix([[(A.T * A).I[0, 0], (A.T * A).I[0, 1]], \
     [(A.T * A).I[1, 0], (A.T * A).I[1, 1]]])
 
+
+def test_issue_2750():
```

Comment:
[This test](https://github.com/sympy/sympy/blob/master/sympy/matrices/expressions/tests/test_matexpr.py#L360) seems to exist already.

Reply:
I'm not sure what the comment means. When I click on the link it just takes me to the code which wasn't added in this PR. Are you being thrown off by what sometimes tricks me: the context of the change showing a def that is actually not part of the change?

Reply:
Yes, I was wondering why GitHub seems to indicate that it was added in this commit.